ibm_db2/assets/configuration/spec.yaml

```
@@ -40,6 +40,14 @@ files:
         value:
           type: integer
           example: 50000
+      - name: security
+        description: |
+          Set this parameter to `ssl` if you want to use the SSL protocol for a connection to the database server.
+          SSL can be used without specifying a certificate for connections to Db2 on Cloud on IBM Cloud. 
+          For those services the required certificate is already a part of the driver package (ibm-db).
```

```suggestion
          Set this parameter to `ssl` to use the SSL protocol for a connection to the database server using the default 
          certificate from the ibm-db driver package. Specifying `tls_cert` is not required when using this option. 
```
Thanks for updating the description! Just a nit.

---

(And then run `ddev validate config ibm_db2 -s` again to sync the config file)

---

Thanks for the suggested change, I incorparated and pushed it.